interpreter/planner.go

```diff
@@ -573,7 +573,6 @@ func (p *planner) idResolver(sel *exprpb.Expr_Select) func(Activation) (ref.Val,
 		case *exprpb.Expr_IdentExpr:
 			ident = op.GetIdentExpr().Name + "." + ident
 			resolvedIdent = true
-			break
```

Comment:
Take a closer look here.  The "break" IMHO is a nicer way to indicate that we're stopping the iteration of the for-loop, rather than making this intention indirect through a boolean.  Since "validIdent" and "resolvedIdent" aren't used past the for-loop, consider eliminating them, have it be a "for {" loop, and for IdentExpr and default just break.

Reply:
The `break` was a mistake on my part as it only applies to the `switch`. I had probably attempted to use it for the `for` loop and then created an infinite loop that I fixed with the boolean switches, but forgot to remove the superfluous `break`.